frontend/consumer-ui/components/Map.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div>
+    <client-only>
+      <l-map class="mini-map" :zoom="13" :center="position">
+        <l-tile-layer url="http://{s}.tile.osm.org/{z}/{x}/{y}.png" />
+        <shop-marker v-for="shop in shops" :key="shop.id" :shop="shop" />
+      </l-map>
+    </client-only>
+  </div>
+</template>
+
+<script>
+import ClientOnly from 'vue-client-only'
+import ShopMarker from './ShopMarker'
+import shops from '~graphql/shops'
+
+export default {
+  apollo: {
+    shops: {
+      query: shops
+    }
+  },
```

Comment:
Here I initially bind the component with its data directly. In storybook's guidelines, they recommend making a presentational component and another component with the data, for the sake of easier testing. @donnerc what best practices do you know on the topic? I would keep it that way for the time being and split up if necessary. I think it would just make our code more complicated for nothing.

Reply:
I've no experience with storybook. But the practice to separate view and data (presentational and data container) is standard. OK to keep it like so for now.